ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -40,8 +44,35 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
               }}
             />
           </div>
+
+          { !isEmpty(questionTemplates) && <div className="mb-3">
+            <label className="form-label" htmlFor="questionTemplate">Question template (optional)</label>
+            <Select
+              options={questionTemplates.map(questionTemplate => {
+                return { label: questionTemplate.name, value: questionTemplate.name }
+              })}
+              isClearable={true}
+              value={selectedQuestionTemplateName}
+              onChange={newValue => {
+                setSelectedQuestionType(undefined)
+                setFreetextMode(false)
+                setSelectedQuestionTemplateName(newValue ? newValue : undefined)
+                setQuestion(unset('type', { name: questionName, questionTemplateName: newValue?.value }) as Question)
+              }}
+            />
+            <p
+              className="form-text"
+              id="questionTemplateDesc"
+            >
+              {'Select a question template to pre-populate the question fields'}
```

Comment:
This doesn't need to be wrapped
```suggestion
              Select a question template to pre-populate the question fields
```